test/lazyload-image.test.ts

```diff
@@ -210,6 +210,38 @@ describe('Lazy load image', () => {
 
             is(result, true);
         });
+
+        it('Should not be visible when image is horizontally in window\'s view, but not in scroll-container\'s', () => {
+            const element = generateElement(800, 0, 1200, 1200);
+            const scrollContainer = generateElement(0, 0, 700, 1200);
+            const result = isVisible(element, 0, _window, scrollContainer);
+
+            is(result, false);
+        });
+
+        it('Should not be visible when image is vertically in window\'s view, but not in scroll-container\'s', () => {
```

Comment:
What do you think of using string template instead of '? Eg.
```
it(`Should not be visible when image is vertically in window's view, but not in scroll-container's`, () => 
```